packages/plugin-react/src/index.ts

```diff
@@ -118,8 +119,16 @@ export default function viteReact(opts: Options = {}): PluginOption[] {
   const viteBabel: Plugin = {
     name: 'vite:react-babel',
     enforce: 'pre',
-    config(_, { mode }) {
+    config(userConfig, { mode }) {
       // Copied from https://github.com/vitejs/vite/blob/4e9bdd4fb3654a9d43917e1cb682d3d2bad25115/packages/vite/src/node/config.ts#L488-L490
```

Comment:
This comment range could be expanded, since we're now copying more of that logic.

Maybe something like: `https://github.com/vitejs/vite/blob/4e9bdd4fb3654a9d43917e1cb682d3d2bad25115/packages/vite/src/node/config.ts#L477-L494`

Reply:
Thank you for your review!